packages/platform/atoms/src/components/ui/button.tsx

```diff
@@ -4,7 +4,12 @@ import * as React from "react";
 
```

Comment:
Could you also have a look at importing the types/React seperately? may be better